content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+(such as/ `jenkins/casc_configs/.dir1/config.yaml`)
+in **any part** of the full path.
+
+* It follows symbolic links for both files and directories.
+
+* The order of traversal does not matter to the final outcome
+because all configuration files that are discovered MUST be supplementary.
+If a file attempts to overwrite configuration values from another file,
+it creates a conflict and raises a `ConfiguratorException`.
+
+=== CasC Configuration and UI Modifications
+
+Configuration for a Jenkins controller should be implemented
+either with CasC or with the UI, but not by both.
+By default, the system allows you to use the UI
+to modify configuration options that were configured by CasC,
+but these modifications are overwritten the next time the controller restarts.
+
+To ensure that configuration changes are not made through the UI,
+install and enable the
+link:https://www.jenkins.io/blog/2020/05/25/read-only-jenkins-announcement/[Read-only Jenkins Configuration]
+plugin.
+When this plugin is installed,
+all configuration fields set by JCasC display as read-only values
+that cannot be modified through the UI.
+
+== For More Information
+
+=== General Information
+
+* link:https://www.youtube.com/watch?v=47D3H1BZi4o[Look Ma! No Hands! -- Manage Jenkins Configuration as Code]
+is a video of the 2018 DevOps World presentation
+that introduced the JCasC feature.
+
+* link:https://www.jenkins.io/blog/2021/05/20/configure-plugins-with-jcasc/[Configure Plugins with JCasC]
+is a blog post with video that demonstrates how to set the JCasC configuration of a plugin.
+
+* link:https://www.youtube.com/watch?v=ANU7jkxbZSM[How to Install Jenkins on CentOS 7 Using Ansible and JCasC]
+is a video presentation with details about using JCasc.
+
+=== Implementation details
+
+Much of the detailed JCasC documentation is provided in the
+link:https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/docs[Github repository].
+
+** link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/IMPLEMENTATION.md[Implementation details]
+
+* The
+link:https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos[demos] 
+directory contains sample _*.yaml_ files for configuring specific Jenkins components and plugins,
+with a _README_ file in each directory that describes the configurations for that component.
+
+* link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/seed-jobs.md[How to create initial "seed" job]
+
+* link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/usageScenarios.md[Usage scenarios]
+
+* link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/configurationReload.md[Triggering Configuration Reload]
+
+* link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/configExport.md[Exporting configurations]
+
+=== Information for Plugin Developers and Maintainers
```

Comment:
The [style guide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments) requests sentence case the "titles" (I interpret that to mean "headings", but that is not a consistent usage throughout the documents, for sure) (optional)

```suggestion
=== Information for plugin developers and maintainers
```